symplyphysics/definitions/damped_harmonic_oscillator_equation.py

```diff
@@ -0,0 +1,139 @@
+from sympy import (
```

Comment:
Does not look like definition. Move it to kinematic.

Reply:
At first I wanted to model it alongside `definitions/harmonic_oscillator_is_second_derivative_equation` so I put it here

Reply:
I see. Well we can leave general equation in definitions, I guess.